script.skin.helper.service/plugin.py

```diff
@@ -46,8 +46,36 @@ def __init__(self):
             elif action == "LAUNCH":
                 xbmcplugin.setResolvedUrl(handle=int(sys.argv[1]), succeeded=False, listitem=xbmcgui.ListItem())
                 path = sys.argv[2].split("&path=")[1]
-                xbmc.executebuiltin("Action(Close)")
                 xbmc.executebuiltin(path)
+            elif action == "FOCUSANDCLICK":
+                #used as workaround to display local media in extendedinfo actorinfo
```

Comment:
Ah OK, I just made an assumption here, didn't actually look at the code.
Well, just think about it if you want it added (it was actually requested a few times to me on the forums). If you like it I can give it a shot and do the PR. You can always decline the PR if you don't like it haha.

For now I've removed the workaround-ish code from skinhelper.